app/src/main/kotlin/com/intive/picover/parties/repository/PartiesRepository.kt

```diff
@@ -20,6 +21,15 @@ class PartiesRepository @Inject constructor(
 			}
 		}
 
+	suspend fun addParty(partyRemote: PartyRemote) =
+		runCatching {
+			partiesReference.push().let { databaseReference ->
+				databaseReference.key?.let { key ->
+					databaseReference.setValue(partyRemote.copy(id = key)).await()
+				} ?: throw Exception()
```

Comment:
I would vote for adding some error message, it could be very simple like `the key from Firebase is null`

Reply:
Good point, I hope the message is descriptive enough :)

Reply:
@BorysChajdas updated